modules/mirror-registry-release-notes.adoc

```diff
@@ -11,6 +11,20 @@ These release notes track the development of the _mirror registry for Red Hat Op
 
 For an overview of the _mirror registry for Red Hat OpenShift_, see xref:../../installing/disconnected_install/installing-mirroring-creating-registry.html#mirror-registry-flags_installing-mirroring-creating-registry[Creating a mirror registry with mirror registry for Red Hat OpenShift].
 
+
+[id="mirror-registry-for-openshift-1-2-7"]
+== Mirror registry for Red Hat OpenShift 1.2.7
+
+_Mirror registry for Red Hat OpenShift_ is now available with Red Hat Quay 3.7.8.
+
+The following advisory is available for the _mirror registry for Red Hat OpenShift_:
+
+* link:https://access.redhat.com/errata/RHBA-2022:6500[RHBA-2022:6500 - mirror registry for Red Hat OpenShift 1.2.7]
+
+=== Bug fixes
+
+* Previously, `getFQDN()` relied on the fully-qualified domain name (FQDN) library to determine its FQDN, and the FQDN library tried to read the `/etc/hosts` folder directly. Consequently, on some {op-system-first} installations with uncommon DNS configurations, the FQDN library would fail to install and abort the installation. With this update, _mirror registry or Red Hat OpenShift_ uses `hostname` to determine the FQDN. As a result, the FQDN library does not fail to install. (link:https://issues.redhat.com/browse/PROJQUAY-4139[*PROJQUAY-4139*])
```

Comment:
```suggestion
* Previously, `getFQDN()` relied on the fully-qualified domain name (FQDN) library to determine its FQDN, and the FQDN library tried to read the `/etc/hosts` folder directly. Consequently, on some {op-system-first} installations with uncommon DNS configurations, the FQDN library would fail to install and abort the installation. With this update, _mirror registry for Red Hat OpenShift_ uses `hostname` to determine the FQDN. As a result, the FQDN library does not fail to install. (link:https://issues.redhat.com/browse/PROJQUAY-4139[*PROJQUAY-4139*])
```
Is this correct?

Reply:
No you are correct. Thanks for catching it.